### ISSUE TITLE
allow upgrades to new release for listed actions

### ISSUE THREAD

**Issue by maxheld83** [r-lib/ghactions#322]
small but weird limitation.

wrote to support:

this is about the new yml-based GitHub actions.

I just tried to cut a new release for an "old" GitHub actions which I had updated for the new yml actions:
https://github.com/maxheld83/pandoc/releases

I also wanted to publish this right away to the marketplace, but GitHub wouldn't let me with this cryptic error message:

conflict.png

I think this was because the *old* action (just the previous release) had this title.

But it should be possible to upgrade existing marketplace entries just with a new release, right?

Only way I could solve this was to delist the old action, and re-list the new one.

Kind of a hassle.

**Comment by bobheadxi** [comment]
ran into this searching for why I can't publish a new package version after migrating to the new yml-based actions 😅did you just have to delist the old action? I currently still can't publish a new version of my action, and I'm not sure if it's because I have to wait longer after delisting or if there's another step I am missing. Thanks!

**Comment by maxheld83** [comment]
yup, I delisted the old action and then listed the new one again, that did the trick.
